tests/functional/demo/test_demo.py

```diff
@@ -157,3 +159,55 @@ def test_workon(database):
     assert params[0]['name'] == '/x'
     assert params[0]['type'] == 'real'
     assert (params[0]['value'] - 34.56789) < 1e-5
+
+
+@pytest.mark.usefixtures("clean_db")
+def test_stress_unique_folder_creation(database, monkeypatch, tmpdir, capfd):
+    """Test integration with a possible framework that needs to create
+    unique directories per trial.
+    """
+    # XXX: return and complete test when there is a way to control random
+    # seed of Oríon
+    how_many = 1000
+    monkeypatch.chdir(os.path.dirname(os.path.abspath(__file__)))
+    process = subprocess.Popen(["orion", "--max-trials={}".format(how_many),
+                                "--pool-size=1",
+                                "--name=lalala",
+                                "--config", "./stress_gradient.yaml",
+                                "./dir_per_trial.py",
+                                "--dir={}".format(str(tmpdir)),
+                                "--other-name~exp.name",
+                                "--name~trial.hash_name",
```

Comment:
We use `~` for both exposed variables and distributions? Wasn't it supposed to be `$` in this case?

Reply:
Nevermind